docs/examples/ftpgetinfo.c

```diff
@@ -26,6 +26,12 @@
 
 #include <curl/curl.h>
 
+#if defined(__MINGW32__)
+#include <inttypes.h>
+#undef CURL_FORMAT_CURL_OFF_T
+#define CURL_FORMAT_CURL_OFF_T PRId64
+#endif
```

Comment:
Is this not something we can/should fix in curl/system.h ?

Reply:
We probably should. But it'd mean to add `inttypes.h` to our public headers, which isn't C89, and I didn't want to make such decision for the sake of fixing these warnings.

Also it's not so simple because this header requires `Visual Studio 2013`, while are still testing for 2010 and even 2008. I'm not sure what we support officially, maybe 2010, which we offer build files for in `projects/Windows/VC10`.

Reply:
<inttypes.h> and the C99 types do actually exist in VS 2010. They were only added for C++ later (via <cstdint> in the std namespace).

Reply:
I based the VC 2013 cutoff version on CMake feature detection results, and it failed to detect `inttypes.h` with VC 2010, e.g.: https://ci.appveyor.com/project/libssh2org/libssh2/builds/46925869/job/wf6g47pff8xg55c3#L84

[ It's possible it's available for VC 2012, too. That version wasn't CI tested, so I went for the safest next version, 2013. I might have made special tests for it, can't remember now. ]

Does it need anything special to use it with 2010?

Reply:
Ah, sorry, you're right! The C99 types in stdint.h were available in VS 2010, but not inttypes.h. I confused the two headers.

Reply:
No worries, thanks for clarifying!

Reply:
Actually the bigger concern is not MSVC here, but mingw. For internal purposes we know that we have mingw-w64, which does have `inttypes.h` support. I have no idea if this is also true for old-mingw.

Let's see how it goes in CI.

Reply:
Updated the public headers, with the caveat that I have no idea about `inttypes.h` support in old/classic/legacy mingw.

Let's see how it goes in CI.